### ISSUE TITLE
[docs] menu strikken-through and warning icon not announced in screen-reader

### ISSUE THREAD

**Issue by SleeplessByte** [expo/expo#39940]
### Summary

<img width="213" height="243" alt="Navigation with some items strikken-through and having an icon" src="https://github.com/user-attachments/assets/c220fec6-7330-418b-a8e6-8721af83d536" />

The visual indicators for old (deprecated?) items are visual only and not announced when using a screen-reader. This makes detecting them much slower, as the page needs to be opened first to see the deprecation message. Please add an accessible label to the icon. Also consider using the `<s>` Strikethrough element but be aware that [that doesn't make it accessible by default](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/s#accessibility).

Please test your documentation against accessibility standards.

### Link to the related documentation page

https://docs.expo.dev/versions/latest/sdk/audio-av/
